### PR TITLE
Feature/react router example

### PR DESCRIPTION
**Pull request by pgerochi**
feat(react-router): integrated route samples

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
 
 export default class App extends React.Component {
   render() {
     return (
-      <div>
-        <h1>It Works!</h1>
-        <p>This React project just works including <span className="redBg">module</span> local styles.</p>
-        <p>Enjoy!</p>
-      </div>
+
```
Comment (alicoding) [inline src/app.jsx]: Indentation looks very off for some reason?

File: src/app.jsx [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+					<div>
+						<ul>
+							<li><Link to="/">Home</Link></li>
+							<li><Link to="/about">About</Link></li>
+							<li><Link to="/sample">Sample</Link></li>
+						</ul>
+
+						<hr/>
+
+						<Route exact path="/" component={Home}/>
```
Comment (alicoding) [inline src/app.jsx]: Could we have these routes definition to be in its own file?

**Comment by pgerochi** [comment]
Don't merge this yet. Let's talk about what i'm doing here.

**Comment by alicoding** [comment]
All of these are awesome!

**Pull request merged**